layers/core_validation.cpp

```diff
@@ -427,8 +429,7 @@ bool CoreChecks::ValidateDrawStateFlags(const CMD_BUFFER_STATE *pCB, const PIPEL
     }
     if (pPipe->topology_at_rasterizer == VK_PRIMITIVE_TOPOLOGY_LINE_LIST ||
         pPipe->topology_at_rasterizer == VK_PRIMITIVE_TOPOLOGY_LINE_STRIP) {
-        const auto *line_state =
-            LvlFindInChain<VkPipelineRasterizationLineStateCreateInfoEXT>(create_info.pRasterizationState->pNext);
+        const auto *line_state = LvlFindInChain<VkPipelineRasterizationLineStateCreateInfoEXT>(rp_state);
```

Comment:
This should probably be `rp_state->pNext`?

Reply:
I think it's still possible for `rp_state` to be null here. I could guard this with a check to see if rp_state is null, but this seemed easier/simpler since `LvlFindInChain` will effectively do the same thing.